lib/eslint.js

```diff
@@ -689,7 +689,7 @@ module.exports = (function() {
                     options,
                     rule;
 
-                if (rules.get(key)) {
+                if (rules.isRulePresent(key)) {
```

Comment:
Do we really need this? If the rule is present  `rules.get` would be a really cheap call because `require` returns the rule function from cache.